neqo-common/src/qlog.rs

```diff
@@ -69,17 +69,20 @@ impl Qlog {
 
     /// Create an enabled `Qlog` configuration.
     ///
+    /// This needs to be called before the connection is used, because otherwise `Qlog`-logging will
+    /// remain disabled (for performance reasons).
```

Comment:
Note that `enabled` is a constructor. With that in mind, I don't understand this comment. How would one call this method after creating the connection?